### PR TITLE
VR patch improvements

### PR DESCRIPTION
**Pull request by TAImatem**

### CODE REVIEW

File: OWML.Patcher/VRFilePatcher.cs [hunk 1]
```diff
@@ -45,60 +45,30 @@ public void Patch()
...
-                }
-            }
-            return false;
+            var zoneString = Encoding.ASCII.GetString(sectorBytes.Skip(startIndex).ToArray());
+            return zoneString.IndexOf(EnabledVRDevice) >= 0;
         }
 
         private byte[] CreatePatchFileBytes(byte[] fileBytes, int patchStartIndex)
         {
             // First byte is the number of elements in the array.
```
Comment (Raicuparta) [inline OWML.Patcher/VRFilePatcher.cs:65]: Does this comment still make sense? Please look at all comments carefully to see if they need to be changed or removed.

Comment (TAImatem) [reply OWML.Patcher/VRFilePatcher.cs]: removed because names of variables look descriptive enough

**Comment by amazingalek** [comment]
yay, more red than green ❤
tested? 🙂

**Comment by TAImatem** [comment]
globalgamemanagers looks exactly the same, can do diff if needed

**Comment by TAImatem** [comment]
Screenshot-comparison

**Pull request merged**